anvil/src/udev.rs

```diff
@@ -244,14 +244,7 @@ pub fn run_udev() {
 
     let gpus = GpuManager::new(GbmGlesBackend::with_factory(|egl_display| {
         let egl_context = EGLContext::new_with_priority(egl_display, ContextPriority::High)?;
-        let disable_color_transform = std::env::var("ANVIL_DISABLE_COLOR_TRANSFORM").is_ok();
```

Comment:
The `README` of anvil also mentions  `ANVIL_DISABLE_COLOR_TRANSFORM`

Reply:
Dropped from the readme.